src/core/execute.c

```diff
@@ -5147,6 +5148,19 @@ int exec_spawn(Unit *unit,
                 }
         }
 
+        if (context->cgroup_nft_family > 0) {
```

Comment:
maybe write this as `context->cgroup_nft_family != NFPROTO_UNSPEC`? i.e. use the symbolic names wherever possible,

Reply:
Problem with `NFPROTO_xxx` is that `#including` `<linux/netfilter.h>` where they are defined causes errors with redefinitions of IP types:
```
In file included from ../src/basic/in-addr-util.h:4,
                 from ../src/basic/ether-addr-util.h:9,
                 from ../src/shared/json.h:12,
                 from ../src/shared/varlink.h:6,
                 from ../src/core/manager.h:18,
                 from ../src/core/emergency-action.h:27,
                 from ../src/core/unit.h:13,
                 from ../src/core/bpf-firewall.h:6,
                 from ../src/core/cgroup.c:12:
/usr/include/netinet/in.h:31:8: error: redefinition of ‘struct in_addr’
   31 | struct in_addr
      |        ^~~~~~~
In file included from /usr/include/linux/netfilter.h:7,
                 from ../src/core/cgroup.c:4:
../src/basic/linux/in.h:89:8: note: originally defined here
   89 | struct in_addr {
      |        ^~~~~~~
```

Reply:
Not applicable anymore